app/src/main/java/com/jxc/app/controllers/UserController.java

```diff
@@ -0,0 +1,48 @@
+package com.jxc.app.controllers;
+
+import com.jxc.app.exceptions.UserNotFoundException;
+import com.jxc.app.models.User;
+import com.jxc.app.services.UserService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/api/v1/users", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+// @RequestMapping(value = "${url.version:/api/v1}" + "${url.users:/users}", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+// I'm still looking for how to use the properties to store the url, using ${} is not working for me
```

Comment:
Peut être qu'il faut une classe abstraite base controller ou une annotation qui étend l'annotation @RestController. Check sur Google. Mais si ça prend du temps ignore en effet, c'est pas hyper important. C'est du nice to have